airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -148,7 +146,7 @@ public static XminStatus getXminStatus(final JdbcDatabase database) throws SQLEx
    */
   public static Map<AirbyteStreamNameNamespacePair, CursorBasedStatus> getCursorBasedSyncStatusForStreams(final JdbcDatabase database,
                                                                                                           final List<ConfiguredAirbyteStream> streams,
-                                                                                                          final StateManager<AirbyteStateMessage, AirbyteStreamState> stateManager,
```

Comment:
StateManager was modified in CDK 0.4.0 to not have any generics. Fixing that here.